src/unstable-temp/DownloadImageModal/OsConfiguration.tsx

```diff
@@ -240,9 +240,9 @@ export const OsConfiguration = ({
 
 	return (
 		<>
-			<Flex mb={3}>
+			<Flex mb={3} ml={-2} mr={-2}>
 				{compatibleDeviceTypes.length > 1 && (
-					<Box flex={3} mr={2}>
+					<Box flex={3} ml={2} mr={2}>
```

Comment:
You can combine all `ml={2} mr={2}` pairs into v :)
```suggestion
					<Box flex={3} mx={2}>
```

Reply:
True 👍